x/nft/types/expected_keepers.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2016-2021 Shanghai Bianjie AI Technology Inc. (licensed under the Apache License, Version 2.0)
+// Modifications Copyright (c) 2021, CRO Protocol Labs ("Crypto.org") (licensed under the Apache License, Version 2.0)
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+// AccountKeeper defines the expected account keeper for query account
+type AccountKeeper interface {
+	GetAccount(ctx sdk.Context, addr sdk.AccAddress) authtypes.AccountI
+}
+
+// BankKeeper defines the expected interface needed to retrieve account balances.
```

Comment:
I guess this only needed for simulations?

Reply:
This is also used in `NewAppModule()`: https://github.com/crypto-org-chain/chain-main/pull/516/files/f828f433f2c80213a04436c6dc999e6e3809e5fd#diff-0fd2887ee17fe532655e8bd2fe7bc4ba896849886b7492dcb0b76824bbb5e95bR106